tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -17,6 +19,7 @@
 using Datadog.Trace.Logging;
 using Datadog.Trace.RemoteConfigurationManagement;
 using Datadog.Trace.Sampling;
+using Action = Datadog.Trace.AppSec.RcmModels.Asm.Action;
```

Comment:
I would not do it for readability, this hides the standard Action delegate. I prefer to do `using Datadog.Trace.AppSec.RcmModels` and then just use`Asm.Action`

Reply:
true thanks, fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3794/commits/109aa12c4886db146ae21e2652c8841105830022